angular-frontend/package.json

```diff
@@ -8,7 +8,8 @@
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
     "test": "ng test",
-    "test-headless": "ng test --watch=false --browsers=ChromeHeadless"
+    "test-headless": "ng test --watch=false --browsers=ChromeHeadless",
+    "serve": "ng serve --host 127.0.0.1"
```

Comment:
**[suggestion]**: should this fold into `start` task? It's a minor configuration change

Reply:
Then the users wouldn't be able to start the app with `localhost` without changing the config. I don't know what is desired.

Reply:
Ok cool, not a blocker.